test/python/qasm3/test_export.py

```diff
@@ -46,8 +46,7 @@ def setUp(self):
             [
```

Comment:
I know `alias_classical_registers` will go away, but it appears that these tests no longer test its behaviour, and I want to check that this is intentional.